src/components/LoadingSpinner.js

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import { keyframes } from 'styled-components';
+
+export default function LoadingSpinner() {
+  return (
+    <SpinnerContainer>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <span className="sr-only">Loading</span>
+    </SpinnerContainer>
```

Comment:
Whatever it is, I would love to see it in action :D

Reply:
Checkout the Vercel Link :) it is a loading spinner with lots of dots :)